web/docs/tutorials/dev-excuses-app/creating-the-project.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Creating the project
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+By now you're already know, [how to create a project and do have Wasp installed](/getting-started.md). So let’s create a new web app project named `ItWaspsOnMyMachine`.
+
+```
+wasp new ItWaspsOnMyMachine
+```
+
+Changing the working directory:
+```
+cd ItWaspsOnMyMachine
+```
+
+Starting the app:
+```
+wasp start
+```
+
+Now your default browser should open up with a simple predefined text message. That’s it! 🥳 For now – the codebase consists of only two files! `main.wasp` is the config file that defines the application’s functionality. And `MainPage.js` is the front-end. Everything else, like `Main.css` can be ignored.
```

Comment:
Aha, we also have Main.css? How is that we ignore it?

Reply:
It's not used by the project (using Tailwind instead) so we are just ignoring it. It can be deleted but decided to avoid that step for simplicity.

Reply:
Aha! I would maybe have step with deleting it -> so the final result is a proper project, I think it is a bit weird that we have file we don't use. And deleting it should be really simple, almost as simple as saying let's ignore it!